ci_framework/roles/edpm_build_images/README.md

```diff
@@ -29,6 +29,7 @@ None
 * `cifmw_edpm_build_images_push_registry_namespace`: (String) Namespace on registry where we want to push container images. Default: `podified-main-centos9`.
 * `cifmw_edpm_build_images_cert_path`: (String) Cert path. Default: `/etc/pki/ca-trust/source/anchors/rh.crt`
 * `cifmw_edpm_build_images_cert_install`: (Boolean) Whether to install cert in the image. Default: false
+* `cifmw_edpm_build_images_base_image`: (String) Base image to package the edpm and ipa qcow2 images into the container images. Default: quay.io/centos/centos:stream9-minimal
```

Comment:
That default value is something that needs maintenance. If someone changes this https://github.com/openstack-k8s-operators/edpm-image-builder/blob/main/Containerfile.image#L1 we need to change the value here. Are we even sure that we are gonna realize that someone changed the value there?
Maybe is better to say, hey, if you provide this value we will override the default one of the image.